torch/testing/_internal/common_device_type.py

```diff
@@ -760,33 +777,26 @@ def _parametrize_test(self, test, generic_cls, device_cls):
             elif self.opinfo_dtypes == OpDTypes.supported:
                 dtypes = op.supported_dtypes(device_cls.device_type)
             elif self.opinfo_dtypes == OpDTypes.any_one:
-                # Arbitrary order
-                dtype_order = (
-                    torch.float32,
-                    torch.float64,
-                    torch.complex64,
-                    torch.complex128,
-                    torch.float16,
-                    torch.bfloat16,
-                    torch.long,
-                    torch.int32,
-                    torch.int16,
-                    torch.int8,
-                    torch.uint8,
-                    torch.bool
-                )
-
                 # Tries to pick a dtype that supports both forward or backward
                 supported = op.supported_dtypes(device_cls.device_type)
                 supported_backward = op.supported_backward_dtypes(device_cls.device_type)
                 supported_both = supported.intersection(supported_backward)
                 dtype_set = supported_both if len(supported_both) > 0 else supported
-                for dtype in dtype_order:
+                for dtype in ANY_DTYPE_ORDER:
                     if dtype in dtype_set:
                         dtypes = {dtype}
                         break
                 else:
                     dtypes = {}
+            elif self.opinfo_dtypes == OpDTypes.any_common_cpu_cuda_one:
```

Comment:
Is this the only place in the codebase where `OpDTypes` are consumed?

Reply:
yes, I believe so